src/control/cmd/dmg/utils_test.go

```diff
@@ -108,20 +113,21 @@ func TestCheckSprint(t *testing.T) {
 				}, MockServers).String(),
 			"1.2.3.4:10000:\n\tPCI Addr:0000:81:00.0 Status:CTL_ERR_APP Error:example application error\n\n1.2.3.5:10001:\n\tPCI Addr:0000:81:00.0 Status:CTL_ERR_APP Error:example application error\n\n",
 		},
-		{
-			NewClientScmResults(
-				[]*ctlpb.ScmModuleResult{
-					{
-						Loc: MockModulePB().Loc,
-						State: &ctlpb.ResponseState{
-							Status: ctlpb.ResponseStatus_CTL_ERR_APP,
-							Error:  "example application error",
-						},
-					},
-				}, MockServers).String(),
-			"1.2.3.4:10000:\n\tModule Location:(socket:4 memctrlr:3 chan:1 pos:2) Status:CTL_ERR_APP Error:example application error\n\n1.2.3.5:10001:\n\tModule Location:(socket:4 memctrlr:3 chan:1 pos:2) Status:CTL_ERR_APP Error:example application error\n\n",
-		},
-		{
+		//"scm operation results": { // currently unused
```

Comment:
Will it be used anytime soon? If not, I'd prefer to delete it.

Reply:
removed in https://github.com/daos-stack/daos/pull/1357